bootstrap/kubeadm/main.go

```diff
@@ -181,6 +188,12 @@ func InitFlags(fs *pflag.FlagSet) {
 // Add RBAC for the authorized diagnostics endpoint.
 // +kubebuilder:rbac:groups=authentication.k8s.io,resources=tokenreviews,verbs=create
 // +kubebuilder:rbac:groups=authorization.k8s.io,resources=subjectaccessreviews,verbs=create
+// ADD CRD RBAC for CRD Migrator.
+// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list;watch
+// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions;customresourcedefinitions/status,verbs=update;patch,resourceNames=kubeadmconfigs.bootstrap.cluster.x-k8s.io
+// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions;customresourcedefinitions/status,verbs=update;patch,resourceNames=kubeadmconfigtemplates.bootstrap.cluster.x-k8s.io
```

Comment:
Could these be combined into a single line with multiple resourceNames?

Reply:
> I wonder if we want to try and condense the CRD rbac by setting the resource name multiple times. It appears the RBAC generator doesn't do any sort of condensing of duplicate rules that only differ by resource name entries

Good point. I wrote the markers like this to ensure it's easy to read them, but it's more important in the roles. I'll see if that helps

Reply:
Works! Nice catch

Reply:
Done & pushed